panel/io/resources.py

```diff
@@ -268,7 +272,7 @@ def resolve_stylesheet(cls, stylesheet: str, attribute: str | None = None):
         The stylesheet definition
     """
     stylesheet = str(stylesheet)
-    if not stylesheet.startswith('http') and attribute and (custom_path:= resolve_custom_path(cls, stylesheet)):
+    if not stylesheet.startswith('http') and attribute and _is_file_path(stylesheet) and (custom_path:= resolve_custom_path(cls, stylesheet)):
```

Comment:
I tried running the below example. But it raised an error because it thought the `_stylesheets` value was a path.

```python
import param
from panel.reactive import ReactiveHTML


class CounterWidget(ReactiveHTML):
    value = param.Integer(default=0)

    _template = """  
    <button id="button_el" class="styled-button" onclick="${script('click_handler')}"></button>  
    """

    _scripts = {
        "render": "self.value()",
        "click_handler": "data.value+=1",
        "value": "button_el.innerHTML = `count is ${data.value}`"
    }
    
    _stylesheets = ["""\
.styled-button {  
    display: inline-block;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-align: center;
    text-decoration: none;
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    cursor: pointer;
    transition: background-color 0.3s;
}  

.styled-button:hover {
    background-color: #45a049;  
}  
"""]  

CounterWidget().servable()
```